app/zandbak.js

```diff
@@ -339,6 +339,12 @@ module.exports = function zandbak(options, backendOptions) {
             return instance;
         },
         exec: (task) => {
+            if (state.filler === emptyFiller) {
```

Comment:
This is a pretty clever use of References in JavaScript, I'd make a mental note to try it in my next project.

Reply:
Have you already passed any course about References in JS? I remember it was a stumbling block ;)

Reply:
I'd say I'm more confident now, at least no longer stumbling but I'm still amazed by all the nifty References tricks people come up with 🐱